tox.ini

```diff
@@ -36,8 +37,21 @@ deps =
     pytest-rerunfailures==9.0
     aioprometheus==20.0.1
     yoti==2.14.0
+    ; Plugin dependencies. We need this
+    ; because we use --no-deps to install the plugins.
+    ; aea_crypto_cosmos/aea_crypto_fetchai
```

Comment:
Why the no deps install?

Reply:
because we want to test the plugins with the `aea` on `develop` (installed by tox during env setup), not from PyPI.

Reply:
True that!